### ISSUE TITLE
ubuntu build fail 

### ISSUE THREAD

**Issue by hunjixin** [sidit77/betrayer#4]
ubuntu 22.04
error log
```
  --> /home/hunjixin/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/betrayer-0.1.0/src/platform/linux/mod.rs:159:77
    |
159 | #[dbus_proxy(interface = "org.kde.StatusNotifierWatcher", assume_defaults = true)]
    |                                                                             ^^^^

error[E0061]: this method takes 1 argument but 2 arguments were supplied
   --> /home/hunjixin/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/betrayer-0.1.0/src/platform/linux/mod.rs:72:29
    |
72  |             conn.executor().spawn(async move {
    |                             ^^^^^
...
101 |             }, "event receiver")
    |              ------------------
    |              | |
    |              | unexpected argument of type `&'static str`
    |              help: remove the extra argument
    |
note: method defined here
   --> /home/hunjixin/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/zbus-3.3.0/src/abstractions/executor.rs:38:12
    |
38  |     pub fn spawn<T: Send + 'static>(
    |            ^^^^^

error[E0433]: failed to resolve: use of undeclared type `StatusNotifierWatcherProxy`
   --> /home/hunjixin/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/betrayer-0.1.0/src/platform/linux/mod.rs:104:21
    |
104 |         let proxy = StatusNotifierWatcherProxy::builder(&conn)
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type `StatusNotifierWatcherProxy`
```

**Comment by sidit77** [comment]
I can't really replicate this issue. I think the issue is that I used `zbus = "3"` and it gets resolved to a different version for you. I replaced it with an exact version on the `main` branch.

**Comment by hunjixin** [comment]
i have found the reason,  as you say, its zbus version, problem, i use this  pakcage https://github.com/frewsxcv/rust-dark-light, but  not knwo why cargo resolve with a old zbus version,  after force set zbus = "3.15.2", build success.

**Comment by hunjixin** [comment]
just update your changes,  zbus is okay, but zbus_macros version fail,  just fixed ```zbus_macros = "4.1.2"``` to resolve

**Comment by hunjixin** [comment]
but i think this reason this dark light use too old version
